tests/foreman/cli/test_import.py

```diff
@@ -1589,7 +1590,7 @@ def test_negative_reimport_chosts(self):
                 )
                 clean_transdata()
 
-    @skip_if_bug_open('bugzilla', 1267224)
```

Comment:
dont replace this one decoration, it's another bug in NEW state
instead add 2nd for bz 1238247